server/graphql/v2/interface/Account.js

```diff
@@ -300,20 +300,52 @@ const accountFieldsDefinition = () => ({
       ...CollectionArgs,
       onlyPublishedUpdates: { type: GraphQLBoolean },
       onlyChangelogUpdates: { type: GraphQLBoolean },
+      orderBy: { type: ChronologicalOrderInput },
```

Comment:
I would make it non-nullable with a default value (createdAt DESC). It's good to have a predictive order for things.

Plus, I'm not sure what happens if this is null line 317, probably a crash when we try to do `orderBy.field`?

Reply:
Yes, agreed it would crash. Seems I've missed this one. I've just corrected it. 🤔 🐨